Added pre-defined criteria: 3-6-9 rules

jojolovearth opened the pull request:
Exclude_sub_HAS_TFR_CASE_IN_N_DAYS
Exclude_sub_TFR_SAID_NO_IN_N_DAYS
Exclude_sub_TFR_SAID_YES_IN_N_DAYS
Exclude_sub_DOWNGRADED_IN_N_DAYS
Exclude_sub_UPGRADED_IN_N_DAYS

jojolovearth committed:
Added predefined criteria: 3-6-9 rules

Exclude_sub_HAS_TFR_CASE_IN_N_DAYS
Exclude_sub_TFR_SAID_NO_IN_N_DAYS
Exclude_sub_TFR_SAID_YES_IN_N_DAYS
Exclude_sub_DOWNGRADED_IN_N_DAYS
Exclude_sub_UPGRADED_IN_N_DAYS

hi-upchen commented:
What's the meaning of `sub`?

jojolovearth committed:
Updated pre-defined criteria: 3-6-9 rules

Criteria:
HAS_TFR_CASE_IN_N_DAYS
TFR_SAID_NO_IN_N_DAYS
TFR_SAID_YES_IN_N_DAYS
DOWNGRADED_IN_N_DAYS
UPGRADED_IN_N_DAYS

Key difference:
also exclude specified sub categories

hi-upchen commented on models/MarketingCloud/JourneyBuilder/DecisionSplitCriteria.js:
```diff
@@ -223,13 +223,13 @@ const DECISION_SPLIT_RULES_BY_SYNC_DE = {
   },
 
   UPGRADED_IN_N_DAYS: {
```
UPGRADED_IN_N_DAYS 不需要過濾  case sub category，請移除  case sub category

hi-upchen commented on models/MarketingCloud/JourneyBuilder/DecisionSplitCriteria.js:
```diff
@@ -223,13 +223,13 @@ const DECISION_SPLIT_RULES_BY_SYNC_DE = {
   DOWNGRADED_IN_N_DAYS: {
-    description: "Last_Successful_Downgrade__c is on or after Today Minus _N_ Days",
-    criteria: "<FilterDefinition><ConditionSet Operator=\"AND\" ConditionSetName=\"Individual Filter Grouping\"><Condition Key=\"Contact_Salesforce_1.Last_Successful_Downgrade__c\" Operator=\"AtOrAfter\" UiMetaData=\"_UI_METADATA_\"><Value><![CDATA[;-;_N_;days]]></Value></Condition></ConditionSet></FilterDefinition>",
+    description: "Last_Successful_Downgrade__c is on or after Today Minus _N_ days AND Sub_Category__c is not equal Welcome Call AND Sub_Category__c is not equal Debit Fail AND Sub_Category__c is not equal Direct Debit Fail AND Sub_Category__c is not equal SDDA Fail AND Sub_Category__c is not equal Expired Card AND Sub_Category__c is not equal Abandon Payment Failure Conversion AND Sub_Category__ ...
```
DOWNGRADED_IN_N_DAYS 不需要過濾 case sub category，請移除 case sub category

jojolovearth committed:
Updated pre-defined criteria: 3-6-9 rules

There are two criteria won't need to exclude specified sub categories, recover to the original condition:

DOWNGRADED_IN_N_DAYS
UPGRADED_IN_N_DAYS
